packages/instantsearch-ui-components/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "instantsearch-ui-components",
+  "version": "0.1.0",
+  "description": "Common UI components for InstantSearch.",
+  "source": "src/index.ts",
+  "types": "dist/es/index.d.ts",
+  "main": "dist/es/index.js",
+  "module": "dist/es/index.js",
+  "type": "module",
+  "exports": {
+    "types": "./dist/es/index.d.ts",
+    "require": "./dist/cjs/index.js",
```

Comment:
if we're not building cjs, we don't have this file right?

Reply:
```suggestion
```

Reply:
Yup, forgot to remove it!